Sources/App/Services/CircleCi+Services.swift

```diff
@@ -418,18 +417,32 @@ extension CircleCi {
             return leftIO(context)(Slack.Response.error(Error.noChannel(from.channelName)))
         }
         let project = projects[index]
+        var rawText = from.text
+        var customChangelog: String?
+        let parts = rawText.components(separatedBy: " customChangelog:\"")
         
-        var parameters = from.text.split(separator: " ").map(String.init).filter({ !$0.isEmpty })
+        if parts.count == 2, let end = parts.last {
```

Comment:
There must be a better solution. Space separated option value is now too specific for just one "customChangelog" and leaves no extendability.

I would search for params which are options (xxx:yyy) where the value is a quoted string and I would replace spaces inside with special character(s). And split by spaces after that. You can replace back special character inside options' values later.

Or someting more flaexible.